AzureMonitor/Scripts/Create-AzDiagPolicy.ps1

```diff
@@ -1857,6 +1862,7 @@ function New-PolicyInitiative
     $InitiativeTemplate = $InitiativeTemplate.Replace("<ParametersGoHere>", $Parameters)
     $InitiativeTemplate = $InitiativeTemplate.Replace("<PolicyDefParams>", $PolicyDefParams)
     $InitiativeTemplate = $InitiativeTemplate.Replace("<ManagementGroupID>", $MGJSONParam)
```

Comment:
I'd name it JSONParam, MG prefix might be confusing

Reply:
Only one more thing: New-PolicyInitiative function is nicely parametrized and after the newest changes, it uses global variable: $ManagementGroupDeployment instead of parameter.

Reply:
Thank you @kwiecek on that catch related to the function - I'll fix the param. Sorry for the delayed response, was on vacation and hardly working.  I do appreciate the ideas and collaboration from you and @dimilider as it creates a stronger product for our customers to utilize so thank you!  Will be making these updates and merging in the next day or so.